### PR TITLE
Detect some cosmetics by barcode

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

**Comment by VaiTon** [comment]
How do we now that products matching that regex are cosmetics? Can you link the source of this info?

File: lib/ProductOpener/SiteQuality_off.pm [hunk 1]
```diff
@@ -828,6 +827,9 @@ sub check_code_gs1_prefixes($) {
 	elsif ($code =~ /^977[0-9]{9,10}$/) {
 		push @{$product_ref->{quality_tags}}, 'gs1-issn-prefix';
 	}
+	elsif ($code =~ /^3600550xxxxxx[0-9]{9,10}$/) {
```
Comment (stephanegigandet) [inline lib/ProductOpener/SiteQuality_off.pm]: the regexp above will match codes that are like "3600550xxxxxx123456789"

```suggestion
	elsif ($code =~ /^3600550[0-9]{6}$/) {
```

**Comment by stephanegigandet** [comment]
> How do we now that products matching that regex are cosmetics? Can you link the source of this info?

https://world.openfoodfacts.org/code/3600550xxxxxx -> the prefix is from a big cosmetics company

File: lib/ProductOpener/DataQualityCommon.pm [hunk 2]
```diff
@@ -167,6 +167,9 @@ sub check_code_gs1_prefixes($) {
 	elsif ($code =~ /^977[0-9]{9,10}$/) {
 		push @{$product_ref->{data_quality_info_tags}}, 'en:gs1-issn-prefix';
 	}
+	elsif ($code =~ /^3600550[0-9]{6}$/) {
+		push @{$product_ref->{quality_tags}}, 'cosmetic-product';
```
Comment (stephanegigandet) [inline lib/ProductOpener/DataQualityCommon.pm]:
```suggestion
		push @{$product_ref->{data_quality_warnings_tags}}, 'cosmetic-product';
```

**Pull request merged**